__test__/crud.test.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-undef */
+const request = require('supertest');
+const app = require('../server/app');
+const { SyllabusModel, connect, disconnect } = require('./db.mocks.js');
```

Comment:
This is the mocked database that's connected to **localhost:27017/test** right?

Reply:
@phucciTr Yep, exactly!